packages/e2e-test/README.md

```diff
@@ -26,19 +29,37 @@ Before running the tests locally, you need to make sure that the environment is
 
 Once the stack is up and running, you can run the following commands to run the tests:
 
-```
+```bash
 npm install
 npm run test
 ```
 
 If you get an error which looks like:
 
-```
+```bash
 Error: Could not find expected browser (chrome) locally.
 ```
 
 Run `npm install` to download the correct Chromium revision (856583), then run `node node_modules/puppeteer/install.js`.
 
+#### Cucumber test individual features
+
+To run a Cucumber test locally we need to have all docker images built and running. We need any of the latest changes.
+
+For example, if we want to test a change in the Next UI, we need to kill the current docker container for the UI. Build
+a new docker image that contains the new changes. And then boot it up from the root of the core directory using `npm run all`.
+
+We can then run this command:
```

Comment:
In general, if there are steps then we should use numbered bullet points to make it easier to follow and read.

```suggestion
To run a Cucumber test locally you need to have all Docker images built and running with the latest changes.

For example, if you want to test a change in the Next UI:

1. Kill the current Docker container for the UI.
1. [Build a new Docker image that contains the new changes](https://github.com/ministryofjustice/bichard7-next-core/blob/main/packages/ui/README.md#building-the-docker-image). 1. Run `npm run all` at the root of the Core repository to boot everything up again.

You can then run this command:
```